evap/evaluation/tests/test_utils.py

```diff
@@ -1,5 +1,6 @@
 from django_webtest import WebTest as DjangoWebTest
 from model_mommy import mommy
+from webtest import AppError
```

Comment:
It was not imported, but used in the file. I don't know why it hasn't crashed before